site/es/lafires-recovery/agenda-setting-findings.mmmd

```diff
@@ -93,7 +93,7 @@ id: findings_how_we_are_responding_list
 title: Oportunidades para crear nuevos programas de apoyo
 nbr_items: 9
 item_1_title:  "Apoyo para la salud mental: crear espacios de recursos comunitarios compartidos"
-item_1_text: Como resultado de los primeros hallazgos y los comentarios de los participantes, el apoyo para la salud mental está disponible para cualquier persona que lo necesite. Hay muchos recursos disponibles ahora, como la asistencia inmediata, el apoyo continuo y la atención para todos los grupos de edad y necesidades de idioma. Esos recursos se pueden encontrar en <a href="https://www.ca.gov/lafires/es/help-for-you/health-and-safety/#Apoyo-de-salud-mental">aquí</a>. 
+item_1_text: Como resultado de los primeros hallazgos y los comentarios de los participantes, el apoyo para la salud mental está disponible para cualquier persona que lo necesite. Hay muchos recursos disponibles ahora, como la asistencia inmediata, el apoyo continuo y la atención para todos los grupos de edad y necesidades de idioma. Puede encontrar esos recursos <a href="https://www.ca.gov/lafires/help-for-you/health-and-safety/#Mental-health-support">aquí</a>.
```

Comment:
We should keep the spanish url here.  href="https://www.ca.gov/lafires/es/help-for-you/health-and-safety/#Apoyo-de-salud-mental

Reply:
Good catch - I'll check the other item_1s as well.